tests/components/fritz/test_button.py

```diff
@@ -77,9 +77,9 @@ async def test_buttons(
         assert button.state != STATE_UNKNOWN
 
 
+@pytest.mark.usefixtures("entity_registry_enabled_by_default")
 async def test_wol_button(
     hass: HomeAssistant,
-    entity_registry_enabled_by_default: None,
```

Comment:
just for my understanding, what is the advantage of using the pytest.usefixture decorator over just using the fixture in the test methods signature?

Reply:
See https://github.com/home-assistant/core/pull/118456
This is now picked up by pylint (in VSCode and when run manually)

Reply:
thx for the context, got it :+1: